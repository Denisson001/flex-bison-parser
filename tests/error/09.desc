Compilation error - invalid assignment rule