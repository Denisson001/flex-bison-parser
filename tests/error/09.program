{
    a = a_s;
    b_s = b;
}