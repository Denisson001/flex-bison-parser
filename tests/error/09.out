Parser: —Åompilation error - syntax error, unexpected _STRING_VAR near line 2
